<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiSettings</class>
 <widget class="QDialog" name="UiSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>4</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NODA Settings</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="locale">
    <locale language="English" country="Germany"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabBar">
       <property name="toolTip">
        <string>x</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabSync">
        <attribute name="title">
         <string>Sync</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>391</width>
           <height>251</height>
          </rect>
         </property>
         <property name="title">
          <string>Synchronium Server</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <widget class="QLabel" name="labelIP">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelPort">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>30</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editIp">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>113</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="editPort">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>50</y>
            <width>121</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="editPassword">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>251</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="labelUsername">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editUsername">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>251</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tabStyle">
        <attribute name="title">
         <string>Style</string>
        </attribute>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Dark mode</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="openFileAct">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="showGeneralOptAct">
   <property name="text">
    <string>Tool Settings</string>
   </property>
   <property name="toolTip">
    <string>Configure CPU</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit FormatX</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="aboutAppAct">
   <property name="text">
    <string>About RCon2</string>
   </property>
  </action>
  <action name="updateAct">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="clearRecentAct">
   <property name="text">
    <string>List Clear</string>
   </property>
  </action>
  <action name="freezeRecentAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>List Freeze</string>
   </property>
  </action>
  <action name="aboutQtAct">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="aboutFBXAct">
   <property name="text">
    <string>About Autodesk® FBX®</string>
   </property>
  </action>
  <action name="showServerSettingsAct">
   <property name="text">
    <string>Server Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
